src/iosrv/module.c

```diff
@@ -253,6 +253,12 @@ dss_module_init_all(uint64_t *mod_facs)
 	uint64_t		 fac;
 	int			 rc = 0;
 
+	/*
+	 * first register global tls accessible to all modules,
+	 * it'll be first initialized and last finialized.
+	 */
+	dss_register_key(&daos_srv_modkey);
```

Comment:
forgot to deregister?

Reply:
actually, we didn't deregister any TLS key in current implementation. Deregister looks not quite necessary to me, but I think we may fix it in other cleanup patch.